### PR TITLE
AirGap Base64 Icon

### PR DESCRIPTION
**Pull request by aj-jester**
* Update Airgap docs to include how to use base64 image for icon.
* Removed Existing Airgap instructions.
* Updated some typos in config.
* Fixed the command in CI/CD Helm Chart.

### CODE REVIEW

File: content/vendor/guides/airgapped-cluster.md [hunk 1]
```diff
@@ -107,35 +107,33 @@ ssh -N -L 8800:${AIRGAP_PRIVATE_IP}:8800 ${JUMPBOX_PUBLIC_IP}
...
 
-### Install via kubectl
+### Converting Icon to Base64
 
-* Once you have your context configured to the airgapped kubernetes cluster, run `kubectl kots install` to install the open source Admin Console components.
+Using `curl` we fetch the image, then use `openssl base64` to encode the image into Base64 and then `pbcopy` to have it copied to your local clipboard.
 
 ```shell
-$ kubectl kots install <your-app-name-and-channel>
+curl -LSs https://sentry-brand.storage.googleapis.com/sentry-glyph-black.png | openssl base64 | pbcopy
```
Comment (dexhorthy) [inline content/vendor/guides/airgapped-cluster.md:119]: amazing

Comment (austinchambers) [reply content/vendor/guides/airgapped-cluster.md]: Maybe we shouldn't use `pbcopy` here, since that's only available on macOS?

**Comment by austinchambers** [comment]
Also, it looks like something broke algolia search: 
```
10:21:20 AM: $ hugo-algolia -s --all
10:21:21 AM: JSON index file was created in public/algolia.json
10:21:21 AM: /opt/build/repo/node_modules/algoliasearch/src/AlgoliaSearchCore.js:54
10:21:21 AM:     throw new errors.AlgoliaSearchError('Please provide an API key. ' + usage);
10:21:21 AM:     ^
10:21:21 AM: AlgoliaSearchError: Please provide an API key. Usage: algoliasearch(applicationID, apiKey, opts)
10:21:21 AM:     at AlgoliaSearchNodeJS.AlgoliaSearchCore (/opt/build/repo/node_modules/algoliasearch/src/AlgoliaSearchCore.js:54:11)
10:21:21 AM:     at AlgoliaSearchNodeJS.AlgoliaSearch (/opt/build/repo/node_modules/algoliasearch/src/AlgoliaSearch.js:11:21)
10:21:21 AM:     at AlgoliaSearchNodeJS.AlgoliaSearchServer (/opt/build/repo/node_modules/algoliasearch/src/server/builds/AlgoliaSearchServer.js:17:17)
10:21:21 AM:     at new AlgoliaSearchNodeJS (/opt/build/repo/node_modules/algoliasearch/src/server/builds/node.js:83:23)
10:21:21 AM:     at algoliasearch (/opt/build/repo/node_modules/algoliasearch/src/server/builds/node.js:68:10)
10:21:21 AM:     at HugoAlgolia.HugoAlgolia.sendIndex (/opt/build/repo/node_modules/hugo-algolia/lib/index.js:184:20)
10:21:21 AM:     at HugoAlgolia.HugoAlgolia.index (/opt/build/repo/node_modules/hugo-algolia/lib/index.js:122:12)
10:21:21 AM:     at Object.<anonymous> (/opt/build/repo/node_modules/hugo-algolia/bin/index.js:23:26)
10:21:21 AM:     at Module._compile (internal/modules/cjs/loader.js:778:30)
10:21:21 AM:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
10:21:21 AM: 
10:21:21 AM: error Command failed with exit code 1.
10:21:21 AM: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
10:21:21 AM: Makefile:35: recipe for target 'index-and-send' failed
10:21:21 AM: make: *** [index-and-send] Error 1
```

**Comment by austinchambers** [comment]
Ok, fixed the algolia issue. Looks like this was because it was an untrusted netlify deploy. Might simplify things to use a branch against this repo instead of a fork.

**Comment by aj-jester** [comment]
@austinchambers Right, I always forget that part.

File: content/vendor/guides/airgapped-cluster.md (hunk 1 above)
Comment (aj-jester) [reply content/vendor/guides/airgapped-cluster.md]: @austinchambers I was thinking about that, what would be the best way to approach this?

**Pull request merged**